tools/test/test.bats

```diff
@@ -55,4 +58,39 @@ setup() {
 @test "can detect chainloop in PATH" {
     run bash -c "source <(./c8l source); is_chainloop_in_path"
     assert_success
+}
+
+# Abats test_tags=bats:focus
+@test "full attestation flow" {
+    export CHAINLOOP_WORKFLOW_NAME="chainloop-labs-tests"
```

Comment:
is this env variable something that the `chainloop_attestation_init` wrapper understands? I am asking because we've removed a similar (or same) env variable from the `init` command itself in the CLI.

Reply:
yes, this is in the wrapper added by @javirln at some point